padd.sh

```diff
@@ -376,15 +361,10 @@ GetNetworkInformation() {
     dhcp_check_box=${check_box_good}
 
     # Is DHCP handling IPv6?
-    if [[ "${DHCP_IPv6}" == "true" ]]; then
```

Comment:
Please do not remove this block. `DHCP_IPv6` is set in `setupVars.conf`. Just add `# shellcheck disable=SC2154` to suppress the warning